hibernate-core/src/main/java/org/hibernate/dialect/OracleServerConfiguration.java

```diff
@@ -45,40 +56,79 @@ public int getDriverMinorVersion() {
 		return driverMinorVersion;
 	}
 
-	public OracleServerConfiguration(boolean autonomous, boolean extended) {
-		this( autonomous, extended, 19, 0 );
+	public OracleServerConfiguration(boolean autonomous, boolean extended, boolean applicationContinuity) {
+		this( autonomous, extended, applicationContinuity, 19, 0 );
```

Comment:
```suggestion
	public OracleServerConfiguration(boolean autonomous, boolean extended) {
		this( autonomous, extended, false, 19, 0 );
```
Since this is an API we should retain the original constructor signature.

Reply:
There's no need for this 3 argument constructor.